source/common/json/json_loader.cc

```diff
@@ -429,16 +429,19 @@ std::string Field::getString(const std::string& name, const std::string& default
   }
 }
 
-std::vector<std::string> Field::getStringArray(const std::string& name) const {
+std::vector<std::string> Field::getStringArray(const std::string& name, bool allow_empty) const {
```

Comment:
nit: optimally it would be nice to add a test case specific to this change in the json tests.